### PR TITLE
feat(ci): add url locale checker

### PR DESCRIPTION
**Pull request by yin1999**
### Description

add url locale checker

### Related issues and pull requests

#7413

### CODE REVIEW

File: files/ru/learn/css/css_layout/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ slug: Learn/CSS/CSS_layout
...
 - [Устаревшие методы вёрстки](/ru/docs/Learn/CSS/CSS_layout/Legacy_Layout_Methods)
   - : Grid-системы - это очень распространённая возможность, используемая в CSS layouts, и до CSS Grid Layout они, как правило, реализовывались с помощью floats или других возможностей вёрстки. Вы представляете свою вёрстку в виде заданного числа столбцов (например, 4 или 6), а затем вы помещаете содержимое в эти воображаемые столбцы. В этом разделе мы рассмотрим, как работают эти старые методы, чтобы вы понимали, как они использовались, если столкнётесь со старыми проектами.
-- [Поддержка старыми браузерами](/en-US/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers)
+- [Поддержка старыми браузерами](/ru/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers)
```
Comment (sashasushko) [inline files/ru/learn/css/css_layout/index.md:44]: @yin1999 thanks for helping. Could you help me to understand the new rules? Now we should place link only for our "own locales"? In that case, it's ok that the page https://developer.mozilla.org/ru/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers isn't existing?

Comment (yin1999) [reply files/ru/learn/css/css_layout/index.md:44]: Yes, yari would automatically falls back to en-US.

image

And doing so can reduce our workload of maintaining documents in the future, without having to update links to other documents after localizing non-existing pages.

For the flaws showing in the preview pages, we have a discuession on it: https://github.com/orgs/mdn/discussions/144.

**Review by wisedog — APPROVED** [review]
LGTM : >

**Review by caugner — APPROVED** [review]

**Comment by yin1999** [comment]
As there is only one url fixed for ru. And the fix looks fine, I'm going to merge this PR.

Thanks for all the reviews :)

**Pull request merged**